src/soundLoop.js

```diff
@@ -65,6 +65,12 @@ define(function (require) {
 
     this._interval = interval || 1;
 
+    // Set the limit of the desiredInterval to zero.
+
+    let desiredInterval = 0;
+
+    SoundLoop._interval = max(desiredInterval, 0.01);
```

Comment:
hrm, `SoundLoop` doesn't exist, instead we have a function called p5.SoundLoop that exists on the p5 object.

We don't want to set _interval on the class, but not on the instance (which is represented by `this`).

That is happening on line 66. So we should do this check on that line.